### PR TITLE
 Fixed: Twig error; #7 

### PR DESCRIPTION
**Pull request by DasLampe**
Fix the twig errors related to #7 

I do not know if this module works, would check this in a few hours.

### CODE REVIEW

File: templates/user.twig [hunk 1]
```diff
@@ -5,12 +5,12 @@
 <h2>{{ pagetitle }}</h2>
 
 {% if userId != false %}
-<p>{{ '{openidProvider:openidProvider:user_page_for}'|trans({'%USERID%', userId|escape('html')}) }}.</p>
```
Comment (tvdijen) [inline templates/user.twig:8]: I'm pretty sure this one is correct

File: templates/user.twig [hunk 2]
```diff
@@ -5,12 +5,12 @@
...
+<p>{{ '{openidProvider:openidProvider:user_page_for}'|trans('%USERID%', userId|escape('html')) }}.</p>
 {% endif %}
 
 {% if loggedInAs is defined %}
 <p><a href="{{ loginURL|escape('html') }}">{{ '{openidProvider:openidProvider:login_view_own_page}'|trans }}</a></p>
-{% elseif ownPage != true) {
+{% elseif ownPage != true %}
```
Comment (tvdijen) [inline templates/user.twig:13]: Thanks for catching this.. For some reason my template unit test fails to run

**Comment by tvdijen** [comment]
It's a pretty old module and it's probably about time we archive it.. Why do you still need to rely on this ancient protocol?

**Comment by DasLampe** [comment]
> Why do you still need to rely on this ancient protocol?

I have some software that supports SSO via OpenID, but no other protocol like SAML2.

**Comment by tvdijen** [comment]
Wow, that software must be a museum piece.. I hope we can make the module work for you, as I have no way of testing it

**Comment by tvdijen** [comment]
I've fixed the first the way it should have been and the latter as is.. Both are in master right now and after I give it a shot to deal with this module (unfamiliar to me) I will tag a new release.

**Pull request closed without merging**